tests/test_rhevm.py

```diff
@@ -0,0 +1,232 @@
+import os
+import pytest
+
+from wrapanapi import RHEVMSystem
+from wrapanapi.exceptions import VMInstanceNotFound
+BAD_NAME_SUFFIX = "-None-shoud-be-found1337"
+
+
+@pytest.fixture(name='rhevm', scope="module")
+def make_rhevm(variables):
+    data = variables.get('rhevm')
+    if not data:
+        pytest.skip('no rhevm data')
+    return RHEVMSystem(**data['system'])
+
+
+@pytest.fixture(name="first_vm_name")
+def fetch_first_first_vm_name(rhevm):
```

Comment:
**QUESTION:** did you really mean _first_first_?

Reply:
fun typo find, i didnt notice since i used function names different from the fixture names to spot accidental uses below